### ISSUE TITLE
convalution havn't activate linear

### ISSUE THREAD

**Issue by piperod** [Tianxiaomo/pytorch-YOLOv4#370]
I am getting this warning error, even though I am using the default cfg for yolov4. not sure if this is crucial for performance. According to the metric provided AP is very very low. What should be the right activation? Should I change something else? I am training on a custom dataset.

**Comment by MattiaMolon** [comment]
I was also wondering what this warning was referring to. It seems like the linear activations are skipped during the construction of the model in:
https://github.com/Tianxiaomo/pytorch-YOLOv4/blob/4ccef0ec8fe984e059378813e33b3740929e0c19/tool/darknet2pytorch.py#L267

I am wondering if this is an error or not. 
If the activation in the yolo.cfg file is linear, shouldn't we do something like this?
```
else:
       model.add_module(("linear{0}".format(conv_id), nn.Linear(filters, filters, bias=False)))
```

Let me know if you managed to solve this

**Comment by Huxwell** [comment]
I am experiencing huge difference in scores (activations, 'confidence', 'probabilities') per bounding box when comparing original darknet scores with scores after onnx conversion.
Wondering if this might be the reason.
@MattiaMolon @piperod has the solution above worked well for you?

**Comment by Huxwell** [comment]
Ok, the solutions seems to be using nn.Identity (a placeholder that essentialy does nothing, but still keep values in the computation graph).

This not only disables the warning, but the scores are now much better.
Before I had some bounding boxes missing totally from onnx runetime, while they were present in darknet and the scores difference was huge (0.1 - 0.4). Now on the same examples the scores difference is only (0.0001-0.2) and the number of bounding boxes is correct.

I

```
                elif activation == 'linear':
                    model.add_module("linear{0}".format(conv_id), nn.Identity(filters, filters, bias=False))
                # based on https://github.com/Tianxiaomo/pytorch-YOLOv4/issues/370
                else:
                    print("convalution havn't activate {}".format(activation))
```

Based on:
https://github.com/pjreddie/darknet/blob/master/src/activations.h
`static inline float linear_activate(float x){return x;}
`
https://pytorch.org/docs/stable/nn.html:
nn.Identity - A placeholder identity operator that is argument-insensitive.
nn.Linear - Applies a linear transformation to the incoming data: y = xA^T + by=xAT+b
